### DISCUSSION TITLE
Снова не работает

### DISCUSSION THREAD

**Discussion by Olezha180** [anticensority/runet-censorship-bypass#344] [Q&A]
Снова не работает

**Comment by ilyaigpetrov** [comment]
Олег, пришлите снимок проблемы, с которой вы сталкиваетесь (ошибка, поломанный интерфейс и т.п.).
Также сообщите: какие конкретно сайты не работают.

**Reply by Olezha180** [reply]
> Олег, пришлите снимок проблемы, с которой вы сталкиваетесь (ошибка, поломанный интерфейс и т.п.). Также сообщите: какие конкретно сайты не работают.

Торрент-сайты, порносайты не открываются

![image](https://github.com/user-attachments/assets/db0176bd-b9a7-4d04-8bdc-c59ec243575f)
![image](https://github.com/user-attachments/assets/86952f3b-07ab-4d6e-a66a-8d264c07e4b6)

**Comment by ilyaigpetrov** [comment]
Какой у вас выбран PAC-скрипт (иконка синей ленты -> вкладка PAC-скрипт)?
Что происходит при попытке нажатия на кнопку "[обновить]" рядом с выбранным PAC-скриптом?
Если возникает ошибка, то пришлите, пожалуйста, её снимок.

**Reply by Olezha180** [reply]
> Какой у вас выбран PAC-скрипт (иконка синей ленты -> вкладка PAC-скрипт)? Что происходит при попытке нажатия на кнопку "[обновить]" рядом с выбранным PAC-скриптом? Если возникает ошибка, то пришлите, пожалуйста, её снимок.

![image](https://github.com/user-attachments/assets/4b918c6d-0451-4a7b-93ff-4fa2718ac5f5)
![image](https://github.com/user-attachments/assets/0b1da797-58f4-4771-b297-339f1045cb89)

**Reply by ilyaigpetrov** [reply]
Следуйте инструкциям из https://github.com/anticensority/runet-censorship-bypass/wiki/Частые-проблемы#не-удалось-скачать-pac-скрипт-с-адресов--httpsantizapretprostovpnorgproxypac%2D.

**Reply by Olezha180** [reply]
> Следуйте инструкциям из [https://github.com/anticensority/runet-censorship-bypass/wiki/Частые-проблемы#не-удалось-скачать-pac-скрипт-с-адресов--httpsantizapretprostovpnorgproxypac%2D](https://github.com/anticensority/runet-censorship-bypass/wiki/%D0%A7%D0%B0%D1%81%D1%82%D1%8B%D0%B5-%D0%BF%D1%80%D0%BE%D0%B1%D0%BB%D0%B5%D0%BC%D1%8B#%D0%BD%D0%B5-%D1%83%D0%B4%D0%B0%D0%BB%D0%BE%D1%81%D1%8C-%D1%81%D0%BA%D0%B0%D1%87%D0%B0%D1%82%D1%8C-pac-%D1%81%D0%BA%D1%80%D0%B8%D0%BF%D1%82-%D1%81-%D0%B0%D0%B4%D1%80%D0%B5%D1%81%D0%BE%D0%B2--httpsantizapretprostovpnorgproxypac%2D).

У меня не открывается вообще файл  https://e.cen.rodeo/proxy.pac

**Reply by ilyaigpetrov** [reply]
Попробуйте сейчас.

**Reply by Olezha180** [reply]
> Попробуйте сейчас.

Открылся файл, что дальше?

**Reply by ilyaigpetrov** [reply]
Обновите PAC-скрипт (кнопка "[обновить]") и попробуйте открыть заблокированные сайты.
Можно прислать снимки ошибок, если таковые уже не были присланы ранее.
